Dockerfile-dcrd

```diff
@@ -4,43 +4,63 @@
 
 FROM ubuntu:xenial
 
-LABEL description="Decred mainnet daemon"
-LABEL version="1.1"
-LABEL maintainer "john@netpurgatory.com"
+LABEL description="Decred mainnet daemon" \
+      version="1.2" \
+      maintainer="john@netpurgatory.com"
 
-ENV DCRD_RELEASE v1.0.5
-
-ENV TERM linux
-ENV USER decred
+ENV DCR_RELEASE="v1.0.5" \
```

Comment:
Ok, will do the change.
I agree, it is the drawback, it is not possible to reuse the env variable as it is not yet available.